pydoctor/model.py

```diff
@@ -554,7 +636,7 @@ def docsources(self) -> Iterator[Documentable]:
         yield self
         if not isinstance(self.parent, Class):
             return
-        for b in self.parent.allbases(include_self=False):
+        for b in self.parent.mro():
```

Comment:
```suggestion
        for b in self.parent.mro()[1:]:
```
A test is required for the inherited doc sources. My code is obviously wrong and the tests still passes :/